lib/envy.ex

```diff
@@ -70,11 +72,18 @@ defmodule Envy do
   defp parse_line(line) do
     [key, value] = line
       |> String.strip
-      |> String.split("=")
+      |> String.split(@key_value_delimeter)
```

Comment:
If we did `|> String.split(@key_value_delimeter, parts: 2)` would that also make the test pass?

Love the extraction of `=` to a module attribute by the way!